### DISCUSSION TITLE
res.text error

### DISCUSSION THREAD

**Discussion by dessyDS** [teamalgoritma/community#365] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/112546949/193214156-574871de-f168-4c24-bcce-344189fee837.png)
Hai, Saya mau bertanya kenapa error dr code diatas? dan juga tidak menambah data di table stations

**Comment by kevbow** [comment, marked as answer]
Dear @dessyDS, 

Apakah Anda menggunakan data stations yang sudah pernah diinput sebelumnya? Jika iya, coba ganti angka di dalam iloc pada cell 85 karena nilai yang dimasukkan ke table dalam database harus bersifat unik dan tidak pernah dimasukkan sebelumnya.

Demikian yang dapat kami sampaikan, terima kasih.

**Reply by dessyDS** [reply]
![image](https://user-images.githubusercontent.com/112546949/193221245-8ad2d6bf-c0b9-4f98-9000-a8668f6faf1b.png)
Hai tetap ga bisa nih, sudah ganti dengan angka lain jg ga bisa, dan ga masuk datanya ke tableplus

**Reply by kevbow** [reply]
Hai,

Coba jalankan kode berikut ini untuk melakukan pengecekan error yang sebenarnya ya bu:
```
data = tuple(stations.iloc[85].fillna('').values)
query_check = f"""
INSERT INTO stations
VALUES {data}
"""

conn.execute(query_check)
```
Jika sudah silakan post kembali errornya disini.

**Reply by dessyDS** [reply]
![image](https://user-images.githubusercontent.com/112546949/193225859-a9850d1a-d73a-48a4-87cb-02c5a3085a21.png)
jika pake kak berikan itu bisa

**Reply by kevbow** [reply]
Jika response yang dikeluarkan angka 200 seharusnya code yang kita masukkan sudah berjalan sehingga kemungkinan ada masalah dengan koneksi dari database, coba ditutup terlebih dulu tableplusnya lalu jalankan kode tersebut untuk melihat apakah koneksi sudah berjalan atau belum.

**Reply by dessyDS** [reply]
sudah ku tutup dan proses tetap tidak bisa :(

**Reply by kevbow** [reply]
Apakah masih menggunakan iloc[85], jika iya sepertinya memang akan error karena Anda menggunakannya di query_check, silakan ubah lagi ke angka random lainnya.
Jika memang belum berhasil akan saya coba bantu di mentoring malam.

**Reply by dessyDS** [reply]
ok, masih blm kak
